src/datastructure.c

```diff
@@ -588,3 +588,41 @@ void _query_set_status(queriesData *query, const enum query_status new_status, c
 	// Update status
 	query->status = new_status;
 }
+
+const char * __attribute__ ((const)) get_query_reply_str(const enum reply_type reply)
```

Comment:
What is this whole part for? Where is the `return "XYZ";` used?

Reply:
It is used here https://github.com/pi-hole/FTL/pull/1287/files#diff-dbb70867a6cb743cbc9b8bb1e9685c340e98cf620ba5dc981f6c593ae992a853R454